### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by nora-weisser**
## Description
Updated README.md:
- included statistics on number of forks, contributors, stars, and issues
- Updated structure of the document
- Added brief information about mentorship programme

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [x] Documentation
- [ ] Other

## Related Issue
N/A

## Motivation and Context
This update makes documentation more informative. This version of README also provides more insights on the repo. 

## Screenshots
Screenshot 2023-10-21 at 19 14 31
Screenshot 2023-10-21 at 19 14 55

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
 The Women Who Code London is a community dedicated to inspiring women to excel in their technology careers. Our events cover various topics around software engineering.
-
 If you're interested in joining the community as a member or volunteer please visit our [website](https://www.womenwhocode.com/london) or join [our slack](https://bit.ly/wwcodelondonslack).
 
-## Contributing
+Check out [Code of Conduct](https://www.womenwhocode.com/codeofconduct/)
+
+# About Mentorship Programme
+
+Mentorship is a fantastic way to get support for your personal growth and career development. Within mentorship relationships,  individuals can experience mutually beneficial outcomes that extend to both their personal and professional lives. Engaging in mentorship offeers a chance to hone communication skills, broaden perspectives, and explore innovative approaches to various situations. Moreover, this collaborative journey allows both mentors and mentees to make significant leap in their career journey.
```
Comment (dricazenck) [inline README.md:15]:
```suggestion
Mentorship is a fantastic way to get support for your personal growth and career development. Within mentorship relationships, individuals can experience mutually beneficial outcomes that extend to their personal and professional lives. Engaging in mentorship offers a chance to hone communication skills, broaden perspectives, and explore innovative approaches to various situations. Moreover, this collaborative journey allows mentors and mentees to make a significant leap in their career journey.
```

File: README.md [hunk 2]
```diff
@@ -28,23 +39,23 @@ We recommend using Bundler to install and run Jekyll. Bundler manages Ruby gem d
...
 
-Run on terminal `bundle exec jekyll serve`
+- Run on terminal `bundle exec jekyll serve`
 
-Access the page on browser: http://127.0.0.1:4000
+- Access the page on browser: http://127.0.0.1:4000
 
 
-### External Links about GitHub Pages
+## External Links about GitHub Pages
```
Comment (dricazenck) [inline README.md:51]: I would move this outside of `How to Run Project Locally` as the last session, `External Resources` for example.

**Review by dricazenck — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,23 +1,34 @@
+[![Contributors][contributors-shield]][contributors-url]
```
Comment (dricazenck) [inline README.md:1]: Really nice!! 🚀

File: README.md (hunk 2 above)
Comment (nora-weisser) [inline README.md:51]:
```suggestion
### External Links about GitHub Pages
```

**Pull request merged**